x-pack/legacy/plugins/reporting/server/core.ts

```diff
@@ -40,36 +48,29 @@ interface ReportingInternalStart {
 }
 
 export class ReportingCore {
-  private pluginSetupDeps?: ReportingInternalSetup;
+  pluginSetupDeps?: ReportingInternalSetup;
```

Comment:
Should still be private, yes?

```suggestion
  private pluginSetupDeps?: ReportingInternalSetup;
```